administrator/modules/mod_guidedtours/src/Helper/GuidedToursHelper.php

```diff
@@ -59,6 +59,9 @@ public function getTours(Registry $params, AdministratorApplication $app)
             $uri = new Uri($item->url);
 
             if ($extension = $uri->getVar('option')) {
+                if ($extension == 'com_categories') {
```

Comment:
Should it not be a strict comparison
```suggestion
                if ($extension === 'com_categories') {
```